functions.php

```diff
@@ -159,3 +183,25 @@ function filter_logo( string $html ) : string {
 	$svg_markup = file_get_contents( __DIR__ . '/assets/logo.svg' );
 	return str_replace( '></', ">$svg_markup</", $html );
 }
+
+/**
+ * Removes parent Customizer controls.
+ *
+ * @since 1.0.1
+ * @param WP_Customize_Manager $wp_customize The customizer manager instance.
+ * @return void
+ */
+function register_customizer( WP_Customize_Manager $wp_customize ) : void {
+	$controls = [
+		'retina_logo',
+		'accent_hue_active',
+		'enable_header_search',
+		'cover_template_separator_1',
+		'cover_template_overlay_background_color',
+		'cover_template_overlay_text_color',
+		'cover_template_overlay_opacity',
```

Comment:
Seems like all that is left is the `header_footer_background_color` control. Is there a good reason to leave this enabled?

Reply:
Fair enough. Resolved in b4da60f6f030cd5da0fb37d19f2e1bd7b73f4482.